packages/styles/scss/components/link-list/_link-list.scss

```diff
@@ -21,8 +21,6 @@
       color: $text-01;
     }
     &__list {
-      overflow: hidden;
```

Comment:
@kennylam I was checking the changelog and this was added for the PR https://github.com/carbon-design-system/ibm-dotcom-library/pull/2452. Just making sure this wasn't added for a particular reason or we're good with this change.

cc: @guilhermelMoraes

Reply:
I thought that too. Another workaround for this problem would be set 1px padding for the list, but I don't know if we can do that

Reply:
Yep, this looks fine to me